src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -580,12 +580,14 @@ public function formAction($id, Request $request)
             $preview_url_deactive = $adminProductWrapper->getPreviewUrl($product, false);
             $preview_url = $adminProductWrapper->getPreviewUrlDeactivate($preview_url_deactive);
         }
+        
+        $shopIdsList = $shopContext->getContextListShopID();
 
         $attributeGroups = $this
             ->getDoctrine()
             ->getManager()
             ->getRepository('PrestaShopBundle:Attribute')
-            ->findByLangAndShop(1, 1);
+            ->findByLangAndShop($legacyContextService->getContext()->language->id, reset($shopIdsList));
```

Comment:
It could be changed to:
`->findByLangAndShop(reset($languages)['id_lang'], reset($shopIdsList));`

Reply:
This could work, indeed